frontend/packages/console-plugin-sdk/src/typings/dashboards.ts

```diff
@@ -13,7 +13,7 @@ import { LazyLoader } from './types';
 namespace ExtensionProperties {
   interface DashboardExtension {
     /** Name of feature flag for this item. */
-    required: string;
+    required: string | string[];
```

Comment:
it would be better to just go with `required: string[]` IMO.

Reply:
actually, nevermind. For 4.3 we will have proper plugin gating and using `string | string[]` will result in smaller change

Reply:
I'd prefer `string | string[]` here to minimize changes in specific plugins.